active/AnimatedIcon/AnimatedIconSpec.md

```diff
@@ -0,0 +1,418 @@
+
+Once the icon file is added to your project you can then add the icon to your navigation view item like you would any other icon type that inherits from IconElement. 
+
+Before: 
+
+XAML
+
+```xml
+<muxc:NavigationView.MenuItems>
+    <muxc:NavigationViewItem Content = "Menu Item1" Icon ="Play" Tag="SamplePage" x:Name = "SamplePageItem">
+</muxc:NavigationView.MenuItems>
+```
+
+After: 
+
+XAML
+
+```xml
+< muxc:NavigationView.MenuItems >
+    < muxc:NavigationViewItem Content = "Menu Item1" Tag="SamplePage" x:Name = "SamplePageItem" >
+        < muxc:NavigationViewItem.Icon >
+            < muxc:AnimatedIcon x:Name = "PlayAnimatedIcon" >
+                < muxc:AnimatedIcon.Source >
+                    < animatedvisuals:PlayIcon />
+                </ muxc:AnimatedIcon.Source>
+            </ muxc:AnimatedIcon >
+    </ muxc:NavigationViewItem.Icon >
+</ muxc:NavigationView.MenuItems >
+```
+
+
+# Add an animated icon to a control that has not been updated for AnimatedIcon 
+
+If you would like to add an animated icon to a control that does not have the default template updated,
+you will need to update the template to include the state changes needed to play the animation segments. 
+
+Here is the XAML for the changes you would need to make to add an AnimatedIcon to a button (button does not have the default template updated). This includes the changes you will need to make to the state transitions for Normal, PointerOver, Pressed, and Disabled. 
+
+This example also shows how to add the fallback static icon that will be used if
+the animated icon cannot be played. In this example we are using a SymbolIcon glyph.
+
+XAML
+
+```xml
+<Button Click="Button_Click"
+        Background="LightGray"
+        Height="100" Widht="80">
+        <AnimatedIcon x:Name ="PlayIcon" >
+            <AnimatedIcon.Source >
+                <animatedvisuals:PlayIcon />
+            </AnimatedIcon.Source>
+            <AnimatedIcon.FallbackIconSource>
+                <SymbolIconSource Symbol="Play"/>  
+            </AnimatedIcon.FallbackIconSource>
+        </AnimatedIcon >
+        <Button.Style>
+            <StyleTargetType="Button">
+                <SetterProperty="Template">
+                    <Setter.Value>
+                        <ControlTemplateTargetType="Button">
+                            <Grid>
+                                <VisualStateManager.VisualStateGroups>
+                                    <VisualStateGroupx:Name="CommonStates">
+                                        <VisualStatex:Name="Normal">
+                                            <VisualState.Setters>
+                                                <Setter Target="Icon.(AnimatedIcon.StateProperty)"Value="Normal"/>
```

Comment:
These need to be just "State", not "StateProperty". And need a space before the "Value=...".